behat.yml.dist

```diff
@@ -3,16 +3,19 @@ default:
     default:
       paths:
         - %paths.base%/tests/features
+        - %paths.base%/modules/oe_paragraphs_media/tests/features
```

Comment:
Since we install the optional module when setting up the dev site let's keep all features in the same place, meaning in 
```
         - %paths.base%/tests/features
```